cpu/stm32_common/include/cpu_conf_stm32_common.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_stm32_common
+ * @{
+ *
+ * @file
+ * @brief           Shared CPU specific configuration for STM32 family
+ *
+ * @author          Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef CPU_CONF_STM32_COMMON_H
+#define CPU_CONF_STM32_COMMON_H
+
+#include "periph_cpu.h"
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F3) || \
+    defined(CPU_FAM_STM32F7) || defined(CPU_FAM_STM32L0) || \
+    defined(CPU_FAM_STM32L4)
+
+static const i2c_timing_param_t timing_params[] = {
+#if defined(CPU_FAM_STM32F0)
+    [ I2C_SPEED_NORMAL ]    = {
```

Comment:
These timings are valid for an input clock wt 48MHz (same for L4 and F7, btw).

The clock requirements should be stated clearly somewhere (alongside the `i2c_conf_t` definition I guess).